### ISSUE TITLE
Instances of insufficient color contrast being missed

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#28]
**Message from a tester:**

I'm also using a couple of other tools to do accessibility scans and something that comes up is the lack of enough contrast in some of the hyperlinks on my site. This gets flagged by WAVE, for example, but Accessibility Checker does not flag the same links. I was curious why that is - is WAVE using a stricter standard?

Here's an example page:
[https://wave.webaim.org/report#/https://webtrainingwheels.com/a-guide-to-moving-wordpress-sites/](https://wave.webaim.org/report#/https://webtrainingwheels.com/a-guide-to-moving-wordpress-sites/)
And some of the contrast errors that WAVE gives but that do not get flagged by Accessibility Checker: [https://jmp.sh/hGu4oL0](https://jmp.sh/hGu4oL0)

Indeed, if you look at her post meta, it's using #777 which fails for small text.  Here's the CSS:
`.meta, .entry .meta {
    font-size: .923em;
    color: #777;
    list-style: none;
    margin: 0 0 10px;
}`

I believe that we have not yet pushed the fix for font size, so I don't think that the em sizing is the problem. Do we know why this is being missed?  If not what do we need to troubleshoot it?

**Comment by SteveJonesDev** [comment]
Closing this issue as it will be fixed with JavaScript scanning. https://github.com/equalizedigital/accessibility-checker/issues/246
